plugin-gradle/README.md

```diff
@@ -510,6 +511,26 @@ spotless {
 
 Spotless uses npm to install necessary packages locally. It runs prettier using [J2V8](https://github.com/eclipsesource/J2V8) internally after that.
 
+
+<a name="antlr4"></a>
+
+## Applying to ANTLR4 source
+
+By default, all ANTLR4 sources matching `src/main/antlr4/**/*.g4` will be formatted. To change this,
```

Comment:
Should mention `src/test/...` as well? (Same in the Maven docs.)

Reply:
Thanks, didn't see that anymore. I started with  only `src/main...`. I'll fix it.